Gruntfile.js

```diff
@@ -388,7 +388,7 @@ module.exports = (grunt) => {
 
     exec: {
       eslint: 'node ./node_modules/eslint/bin/eslint.js --cache "**/*.{js,vue}"',
-      flow: '"./node_modules/.bin/flow" --quiet || echo The Flow check failed!',
+      flow: '"./node_modules/.bin/flow" --quiet',
```

Comment:
The `echo` part made it swallow the error, which made it possible to run the server despite Flow errors. With this change, an error will make `grunt` stop. The practical consequence of this is that CI flows will fail in case of Flow errors.